src/common/src/session_config/mod.rs

```diff
@@ -134,7 +134,7 @@ pub struct ConfigMap {
     streaming_enable_bushy_join: bool,
 
     /// Enable arrangement backfill for streaming queries. Defaults to false.
-    #[parameter(default = false)]
+    #[parameter(default = true)]
     streaming_enable_arrangement_backfill: bool,
```

Comment:
I'd suggest changing the default value in a separate PR.

Reply:
Temporary change, just to test arrangement backfill. It will be reverted.